### PR TITLE
Reduce commercial CSS complecting: top slot

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Refactoring commercial CSS looks like a party going all wrong:

giphy 2

... but if we're looking to use [Atomic CSS](https://github.com/guardian/frontend/pull/13549), we'd better start somewhere. One thing we can do is reduce the "cyclomatic complexity" in our selectors. This will improve the overall _aesthetic_ of our stylesheet but also stop the sheer number of times we need to override styles in specificity wars.

I start slow, with the top slot, and am voluntarily spartan about removing styles for now—it will become easier as all selectors converge towards the smallest specificity.

cc @guardian/commercial-dev @sndrs

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
(This also fixes the FOOSA which happens for some of our home-fledged creatives)

*FOUSA = Flash Of Overlapping Sticky Ad

File: common/test/views/support/CommercialTest.scala [hunk 1]
```diff
@@ -42,13 +42,4 @@ class CommercialTest extends FlatSpec with Matchers with OptionValues with Befor
   // they should "be responsive for 88x70 ad on US business front" in {
   //   topAboveNavSlot.cssClasses(metaDataFromId("us/business"), defaultEdition, Seq(responsiveSize)) should endWith("top-banner-ad-container--responsive")
   // }
-
-  they should "be default for any other page" in {
```
Comment (rich-nguyen) [inline common/test/views/support/CommercialTest.scala]: Should this test be removed? If the test was edited to check `endWith("js-top-banner-desktop")`, would it pass?

File: static/src/javascripts/projects/common/modules/commercial/dfp/create-slot.js [hunk 2]
```diff
@@ -94,16 +94,6 @@ define([
                     adSizes.fluid250,
                     adSizes.fabric,
                     adSizes.fluid
-                ),
-                desktop: compile(
-                    adSizes.outOfPage,
-                    adSizes.fluid250,
-                    adSizes.leaderboard,
-                    adSizes.cascade,
-                    adSizes.superHeader,
```
Comment (rich-nguyen) [inline static/src/javascripts/projects/common/modules/commercial/dfp/create-slot.js]: can you also remove some of the entries from `ad-sizes.js`, like 'superHeader'?

**Comment by rich-nguyen** [comment]
👍  nice

File: static/src/javascripts/projects/common/modules/commercial/dfp/create-slot.js (hunk 2 above)
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/commercial/dfp/create-slot.js]: Yep, nice catch!

File: common/test/views/support/CommercialTest.scala (hunk 1 above)
Comment (regiskuckaertz) [inline common/test/views/support/CommercialTest.scala]: Makes sense, yes.

**Pull request merged**